debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -174,4 +193,7 @@ protected static int validateServerNameIsDifferentFromHistoryTopicName(Configura
         return 0;
     }
 
+    protected SourceInfoStructMaker<? extends AbstractSourceInfo> getDefaultSourceInfoStructMaker() {
```

Comment:
Should be abstract actually?

Reply:
@gunnarmorling It will be - when all connectors will support struct maker